fish/functions/update.fish

```diff
@@ -63,7 +63,7 @@ function update --description 'Runs the varius upgrade commands'
 
   log "Cleaning up"
   if test -n "$OUTDATED"
-    brew prune
+    brew cleanup --prune-prefix
     brew cleanup -s
```

Comment:
WDYT about combining these? `brew cleanup -s --prune-prefix`?

Reply:
I do not mind at all, I'm not familiar with what the flags really do, so I just kept it simple.